spec/spec_helper.rb

```diff
@@ -1,6 +1,14 @@
 # frozen_string_literal: true
 
+require_relative 'support/simplecov'
+
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 
 require 'hets-rabbitmq-wrapper'
-require_relative 'support/simplecov'
+require 'bunny-mock'
+
+RSpec.configure do |config|
+  config.before(:each) do
```

Comment:
Is it necessary to do this before `:each` (repeatedly before every single example) or is it okay to do it before `:suite` (before the first example only)?